### ISSUE TITLE
Metroid Prime Remastered issues after mainline 1479

### ISSUE THREAD

**Issue by catmality** [yuzu-emu/yuzu#11023]
### Is there an existing issue for this?

- [X] I have searched the existing issues

### Affected Build(s)

Mainline 1479 and above

### Description of Issue

Yuzu versions 1479 and above have had two strange anomalies pop up in Prime Remastered. 

These issues only seem to happen in Exclusive Fullscreen
1. Loading textures seems to cause a lot more 'colorful' textures appear, as can be seen in certain rooms and going back and forth in between.
2. Opening the map/inventory screen sometimes reveals glitched textures. Closing the map/inventory screen seems to then 'refresh' all the loaded textures in the room, causing a myriad of incorrect and colorful textures for a short moment.

Also a couple other things:
-Mainline 1483 broke something in the game and the lighting since then hasn't been rendering properly. Notably, enemies and the arm cannon are pitched completely black when there should be lighting. I do see that EA 3737 put out a fix for this.
-Samus's visor doesn't work. In cutscenes her face is always completely revealed (instead of the green visor that covers it slightly) and she's ghostly white. This issue has been in all the mainline builds.

Youtube link of what it looks like for me: https://youtu.be/bOhBdZh1A7s

### Expected Behavior

1. Going between rooms shouldn't be as colorful. In older versions the textures would look a little more darker before properly loading.
2. The map screen was working fine before - it should not be producing wrong textures or making the game refresh. (Okay, turns out the map/inventory screen is still a little busted in older versions, but in my testing, versions before 1479 had this happen less often?) 

Youtube video of what it was like before: https://youtu.be/AORUz2cZ4Wk

### Reproduction Steps

1. Run Metroid Prime Remastered in Exclusive Fullscreen mode.
2. It's most notable in the room Hive Totem and the reward room past it. You can use my save file to get there easily. (Load file 1, go straight out of the room and follow the hallways. The big chamber is the Hive Totem room. The room in the back is the reward room.) [MP save.zip](https://github.com/yuzu-emu/yuzu/files/11952169/MP.save.zip)
3. You can test problem 2 easily by just opening the map screen with the pause button. Closing it should reproduce this issue.
4. You can test problem 1 by going back and forth the Hive Totem room and the reward room. Notably, the door when you enter the reward room tends to turn bright yellow, and going out of it the floor surrounding you will be pink-ish. Going back in, you can see the door when it fully opens at the bottom turn colorful.

### Log File

[yuzu_log.txt](https://github.com/yuzu-emu/yuzu/files/11952248/yuzu_log.txt)

### System Configuration

CPU: AMD Ryzen 7 2700x 8 core
GPU/Driver: AMD Radeon RX 7900 XTX / 23.5.2 (latest)
RAM: 32 GB @ 3200MHz
OS: Windows 10 Home 22H2

**Comment by Squall-Leonhart** [comment]
https://github.com/yuzu-emu/yuzu/pull/11012

**Comment by mikegonzalez2k** [comment]
The Texture issue seems fixed as of 1490.  There are new issues buffering issues however.
#11053

**Comment by catmality** [comment]
> 

yeah, I did notice this as of recent - was fun blowing up some boxes or an enemy and the game hard dropping frames for a solid 3 or so seconds. seems to happen inconsistently. but yeah, the lighting issue was fixed.
